src/sass/_footer.scss

```diff
@@ -0,0 +1,5 @@
+.footer {
+  background-color: black;
```

Comment:
I think you can extend these properties by Bootstrap classes like bg-dark and such :)